vtexdocs/devportal
# Feat/search-page

#### What is the purpose of this pull request?

Add the entire results page.

#### How should this be manually tested?

Go to [this page](https://deploy-preview-331--elated-hoover-5c29bf.netlify.app/) and try to type something in the search bar. Type Enter or click on the option "See all results" (If it isn't appearing, it is because the response returns 7 or fewer hits.)

Once on the search page, try to navigate by pages, select filters and check if the behavior works without problems.

#### Screenshots or example usage

https://user-images.githubusercontent.com/42784961/232628381-50a2a7b5-b114-459a-b4e5-41e7be11e98a.mp4

#### Types of changes
